lib/game_parser.rb

```diff
@@ -7,4 +9,21 @@ def initialize(file)
   def get_first_line
     File.open(@file, &:readline)
   end
+
+  def get_log_metadata
+    path = get_file_path
+    lines = get_number_of_lines
+    metadata = {path => {"lines" => lines}}
+    JSON.pretty_generate(metadata)
+  end
+
+  def get_number_of_lines
+    File.readlines(@file).length
+  end
+
+  def get_file_path
+    File.expand_path(@file)
+  end
+
+  private :get_number_of_lines, :get_file_path
```

Comment:
Geralmente é mais comum os métodos privados serem declarados assim:


```suggestion
  private
  
  def get_number_of_lines
    File.readlines(@file).length
  end

  def get_file_path
    File.expand_path(@file)
  end
```